src/instruments/ecam-page-template/template.js

```diff
@@ -36,36 +36,40 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-/* global BaseInstrument */
-/* global registerInstrument */
+/* global Airliners */
 
-// eslint-disable-next-line camelcase
-class A32NX_PAGE_NAME_Logic extends Airliners.EICASTemplateElement {
-    get templateID() {
-        return 'A32NX_PAGE_NAME_TEMPLATE';
-    }
+let A32NX_PAGE_NAME;
+((A32NX_PAGE_NAME) => {
+    // eslint-disable-next-line camelcase
+    class A32NX_PAGE_NAME_Logic extends Airliners.EICASTemplateElement {
+        get templateID() {
+            return 'A32NX_PAGE_NAME_TEMPLATE';
+        }
 
-    connectedCallback() {
-        super.connectedCallback();
+        connectedCallback() {
+            super.connectedCallback();
 
-        // This is big hack, see `template.html`.
-        {
-            const code = document.getElementById('A32NX_BUNDLED_STYLE').innerHTML;
-            const style = document.createElement('style');
-            style.innerHTML = code;
-            document.head.appendChild(style);
-        }
-        {
-            const code = document.getElementById('A32NX_BUNDLED_LOGIC').innerHTML;
-            const script = document.createElement('script');
-            script.innerHTML = code;
-            document.body.appendChild(script);
+            // This is big hack, see `template.html`.
+            {
+                const code = document.getElementById('A32NX_BUNDLED_STYLE').innerHTML;
+                const style = document.createElement('style');
+                style.innerHTML = code;
+                document.head.appendChild(style);
+            }
+            {
+                const code = document.getElementById('A32NX_BUNDLED_LOGIC').innerHTML;
+                const script = document.createElement('script');
+                script.innerHTML = code;
+                document.body.appendChild(script);
+            }
         }
-    }
 
-    update(_deltaTime) {
-        this.dispatchEvent(new CustomEvent('update', { detail: this.deltaTime }));
+        update(_deltaTime) {
+            this.dispatchEvent(new CustomEvent('update', { detail: this.deltaTime }));
+        }
     }
-}
+    // eslint-disable-next-line camelcase
+    A32NX_PAGE_NAME.Page = A32NX_PAGE_NAME_Logic;
+})(A32NX_PAGE_NAME || (A32NX_PAGE_NAME = {}));
```

Comment:
I don't get why this has to be an IIFE.

Reply:
Can't say I understand why. Just following every other page and figured my JS knowledge is fairly weak.

Reply:
@Benjozork what is your thought on the matter?

Reply:
IIFE gone, seems to work just fine as expected. Thanks for the feedback, and now I'm slightly less of a JS noob. 👍